sdk/python/feast/feature_view.py

```diff
@@ -83,7 +99,9 @@ def __init__(
             self.ttl = ttl
 
         self.online = online
-        self.input = input
+        self.input = _input
+        self.batch_source = _input
+        self.stream_source = stream_source
```

Comment:
Should we validate that a batch source exists for all stream sources?

Reply:
yes

Reply:
This is already happening in line 79-80